rst/index.rst

```diff
@@ -1,23 +1,23 @@
-FAQ pagoPA (BOZZA)
+FAQ pagoPA
 ==================
 
-Nota: Di seguito trovate la documentazione originale da cui siamo partiti per il lavoro di conversione al nuovo formato RST:
+Di seguito trovate la documentazione originale da cui siamo partiti per il lavoro di conversione al nuovo formato RST:
```

Comment:
A chi ci rivolgiamo? Chi è "voi"? Chi siamo "noi"? Conviene forse utilizzare espressioni più neutre in questo caso, come, ad esempio, "Qui è possibile trovare la documentazione originale utilizzata per il lavoro ..."?

Reply:
Ok! In questo primo round di revisione ci si è concentrati su: compilazione sphinx senza errori, tabelle, immagini, note. I contenuti nel dettaglio faranno parte di un altro round successivo.

Reply:
Benissimo! Per me si può fare il merge.